src/statespace/SE2.cpp

```diff
@@ -66,25 +66,35 @@ void SE2::discrete_state_to_continuous(
     const StateSpace::State& _state,
     aikido::statespace::StateSpace::State* _continuous_state) const {
     const State state = static_cast<const State&>(_state);
-    
-    Eigen::VectorXd state_log(3);
+    Eigen::Vector3d state_log;
     state_log.head<2>() =
         discrete_position_to_continuous(Eigen::Vector2i(state.x, state.y));
     state_log[2] =
         discrete_angle_to_continuous(state.theta);
-    m_statespace->expMap(state_log, _continuous_state);
+    double angle = state_log(2);
```

Comment:
This is redundant...
just do `const double angle = discrete_angle_to_continuous(state.theta)`